src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1598,7 +1598,7 @@ internal enum ErrorCode
         ERR_PossibleAsyncIteratorWithoutYieldOrAwait = 8420,
         ERR_StaticLocalFunctionCannotCaptureVariable = 8421,
         ERR_StaticLocalFunctionCannotCaptureThis = 8422,
-
+        ERR_AttributeNotOnEventAccessor = 8423,
```

Comment:
I am not sure how new error codes are picked. I am also not sure whether this should go with C# 1 errors, or C# 8, as it's just a different name/message for what is essentially an old error.

Reply:
This number (in the C# 8.0 range) is fine.

---
In reply to: [265474149](https://github.com/dotnet/roslyn/pull/32571#discussion_r265474149) [](ancestors = 265474149)